Makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: all configure build repl nix-shell update-nix
+
+all:
+	-@echo "Targets:  update-nix"
+	-@echo "  configure:  cabal configure"
+	-@echo "  build:      cabal build"
+	-@echo "  repl:       cabal repl"
+	-@echo "  nix-build:  build via nix"
+	-@echo "  nix-shell:  open a nix shell"
+	-@echo "  update-nix: update the nix files"
+
+configure:
+	cabal configure
+
+build:
+	cabal build -j
+
+repl:
+	cabal repl
+
+nix-build: nix/capnp.nix
+	-@rm -f result*
+	nix-build --no-out-link shell.nix -Q -j 8
```

Comment:
Should probably use `nproc` instead of hard-coding 8.

I'm also not sure how I feel about this makefile in general; most of the targets are just trivial wrappers around a command that's just as intuitive (and probably more familiar); maybe just cut those out?